src/ResourceManager/SignalR/Commands.SignalR/Cmdlets/RemoveAzureRmSignalR.cs

```diff
@@ -0,0 +1,96 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.SignalR.Models;
+using Microsoft.Azure.Commands.SignalR.Properties;
+using Microsoft.Azure.Management.SignalR;
+using System;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.SignalR.Cmdlets
+{
+    [Cmdlet(VerbsCommon.Remove, SignalRNoun, SupportsShouldProcess = true, DefaultParameterSetName = ResourceGroupParameterSet)]
+    [OutputType(typeof(bool))]
+    public class RemoveAzureRmSignalR : SignalRCmdletBase, IWithInputObject, IWithResourceId
+    {
+        [Parameter(Position = 0,
+            Mandatory = false,
+            ParameterSetName = ResourceGroupParameterSet,
+            HelpMessage = "The resource group name. The default one will be used if not specified.")]
+        [ResourceGroupCompleter()]
+        [ValidateNotNullOrEmpty]
+        public override string ResourceGroupName { get; set; }
+
+        [Parameter(Position = 1,
+            Mandatory = true,
+            ParameterSetName = ResourceGroupParameterSet,
+            HelpMessage = "The SignalR service name.")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
```

Comment:
Position: if the parameter **Name**  is mandatory should it go before the **ResourceGroupName**  parameter?